lib/stm32/g0/Makefile

```diff
@@ -39,6 +39,7 @@ OBJS += flash.o flash_common_all.o
 OBJS += gpio_common_all.o gpio_common_f0234.o 
 OBJS += pwr.o
 OBJS += rcc.o rcc_common_all.o
+OBJS += iwdg_common_all.o
```

Comment:
this is actually alphabetically sorted now, let's try and keep it that way, easier to find things.